internal/app/server/repository/repository.go

```diff
@@ -19,6 +19,11 @@ type Book struct {
 }
 
 func NewFileBookRepository(filePath string) *Book {
+	err := os.MkdirAll(filePath, 0755)
+	if err != nil {
+		panic(err)
```

Comment:
it better be done the same way as with DB (if it panics, DB version should panic to, if it returns error, both versions should do so)